Examples/Scripts/SetUpSPECIES.sh

```diff
@@ -0,0 +1,341 @@
+    T1wTemplate2mm="${HCPPIPEDIR_Templates}/NHP_NNP/${BrainTemplate}/MNINonLinear/${BrainTemplate}_T1w_1.6mm.nii.gz"  
+    T1wTemplate2mmBrain="${HCPPIPEDIR_Templates}/NHP_NNP/${BrainTemplate}/MNINonLinear/${BrainTemplate}_T1w_1.6mm_brain.nii.gz" 
+    T2wTemplate="${HCPPIPEDIR_Templates}/NHP_NNP/${BrainTemplate}/MNINonLinear/${BrainTemplate}_T2w_${StrucRes}mm.nii.gz"  
+    T2wTemplate2mmBrain="${HCPPIPEDIR_Templates}/NHP_NNP/${BrainTemplate}/MNINonLinear/${BrainTemplate}_T2w_1.6mm_brain.nii.gz" 
+    T2wTemplateBrain="${HCPPIPEDIR_Templates}/NHP_NNP/${BrainTemplate}/MNINonLinear/${BrainTemplate}_T2w_${StrucRes}mm_brain.nii.gz" 
+    T2wTemplate2mm="${HCPPIPEDIR_Templates}/NHP_NNP/${BrainTemplate}/MNINonLinear/${BrainTemplate}_T2w_1.6mm.nii.gz" 
+    TemplateMask="${HCPPIPEDIR_Templates}/NHP_NNP/${BrainTemplate}/MNINonLinear/${BrainTemplate}_T1w_${StrucRes}mm_brain_mask.nii.gz" 
+    Template2mmMask="${HCPPIPEDIR_Templates}/NHP_NNP/${BrainTemplate}/MNINonLinear/${BrainTemplate}_T1w_1.6mm_brain_mask.nii.gz" 
+
+    #PostFreeSurferPipeLineBatch.sh
+    MyelinMappingFWHM="4" # based on median cortical thickenss of owl
+    SurfaceSmoothingFWHM="4" # 4 by default 
+    CorrectionSigma=6
+    SurfaceAtlasDIR="${HCPPIPEDIR_Templates}/NHP_NNP/${BrainTemplate}/standard_mesh_atlases"
+    GrayordinatesSpaceDIR="${HCPPIPEDIR_Templates}/NHP_NNP/${BrainTemplate}/standard_mesh_atlases"
+    ReferenceMyelinMaps="${HCPPIPEDIR_Templates}/NHP_NNP/${BrainTemplate}/standard_mesh_atlases/ChimpYerkes29.MyelinMap_BC.164k_fs_LR.dscalar.nii"
+    LowResMeshes="32@20" #Needs to match what is in PostFreeSurfer
+    FinalfMRIResolution="1.6" #Needs to match what is in fMRIVolume
+    SmoothingFWHM="1.6" #Recommended to be roughly the voxel size
+    GrayordinatesResolution="1.6" #should be either 1 (7T) or 2 (3T) for human. 
+
+elif [[ "$SPECIES" == *Macaque* ]] ; then
+
+    BrainScaleFactor="2"
+    CorticalScaleFactor="3"
+
+    #PreFreeSurferPipeLineBatch.sh
+    BrainSize="60"               # BrainSize in mm, distance bewteen top of FOV and bottom of brain
+
+    FNIRTConfig="${HCPPIPEDIR_Config}/T1_2_NHP_NNP_Macaque_1mm.cnf" #FNIRT 2mm T1w Config  High resolution warping, optimized regularization
+    TopupConfig="${HCPPIPEDIR_Config}/b02b0_macaque_fMRI.cnf" #Config for topup or "NONE" if not used
+    BiasFieldSmoothingSigma="3.5"
+
+
+    if [[ "$SPECIES" == *Mac30BS* ]] ; then     # Rhesus and Cyno hybrid template
+
+        betcenter="48,56,51"         # comma separated voxel coordinates in T1wTemplate2mm
+        betradius="35"
+        betbiasfieldcor="TRUE"
+        betfraction="0.2"
+        bettop2center="30"           # distance in mm from the top of FOV to the center of brain in robustroi
+
+        BrainTemplate="Mac30BS"
+        T1wTemplate="${HCPPIPEDIR_Templates}/NHP_NNP/${BrainTemplate}/MNINonLinear/${BrainTemplate}_T1w_restore.nii.gz" 
```

Comment:
If Mac30BS will never have another resolution, then maybe we should check that `$StrucRes` equals whatever the resolution is in this template.  Otherwise, we may want to put `$StrucRes` into the naming of these templates for consistency.